.github/workflows/integration.yml

```diff
@@ -24,18 +24,11 @@ jobs:
           python-version: ${{ matrix.python-version }}
           cache: "pip"
 
-      - name: Upgrade pip
-        run: |
-          pip install --constraint=.github/workflows/constraints.txt pip
-          pip --version
-
-      - name: Install Poetry
-        run: |
-          pipx install --pip-args=--constraint=.github/workflows/poetry-constraints.txt poetry
-          poetry --version
+      - name: Install UV
+        uses: astral-sh/setup-uv@v5
```

Comment:
Should we set the version of `uv` to use like we did for `poetry`?

The documentation has [pointers](https://github.com/astral-sh/setup-uv) on how to do this

Reply:
Good point, it is a good idea. Thanks!

Reply:
Done.